src/e2e-tests/page-objects/smartthings.page.js

```diff
@@ -3,9 +3,269 @@ import Page from './page';
 import MenuComponent from './menu.component';
 
 class SmartThingsPage extends Page {
```

Comment:
Since this is big, can you please create components for each tab?
As an example you can use the MenuComponent and how JS composition is used

Reply:
Updated structure.